src/main/java/frc/robot/commands/ClawSwitch.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Claw;
+
+public class ClawSwitch extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final Claw m_subsystem;
+
+  private boolean isClosed;
+  private boolean isOpen;
+  private boolean startClosed;
+
+  public ClawSwitch(Claw subsystem) {
+    m_subsystem = subsystem;
+    System.out.println("Ignore a warning about not using m_subsystem:" + m_subsystem);
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(subsystem);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_subsystem.checkLimits();
+    isClosed = m_subsystem.queryClosed();
+    isOpen = m_subsystem.queryOpen();
+
```

Comment:
What happens if the driver doesn't hold down the button until one of the limits is set.
I think you'll end up ttaking the 'else' in the 'if' below.

I think as it is, if you're not all the way closed or all the way open then you'll start closing.

Is that intended?  Alternatively we could just have it switch direction (like a garage door).